internal/repository/account_repo_psql.go

```diff
@@ -0,0 +1,95 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/gopay/internal/models"
+)
+
+const (
+	createQ = `
+	INSERT INTO accounts 
+	(name, last_name) 
+	VALUES ($1, $2) 
+	RETURNING account_id
+	`
+
+	findAllQ = `
+	SELECT * 
+	FROM accounts
+	`
+
+	findOneQ = `
+	SELECT *
+	FROM accounts
+	WHERE account_id = $1
+	`
+)
+
+type AccountRepoPsql interface {
+	FindAll(ctx context.Context) ([]models.Account, error)
+	FindOne(ctx context.Context, id string) (models.Account, error)
+	Create(ctx context.Context, name string, lastname string) (string, error)
+}
+
+var _ AccountRepoPsql = (*accountRepoPsqlImpl)(nil)
+
+type accountRepoPsqlImpl struct {
+	psql *sql.DB
+}
+
+func NewAccountRepoPsql(db *sql.DB) *accountRepoPsqlImpl {
+	return &accountRepoPsqlImpl{
+		psql: db,
+	}
+}
+
+func (r *accountRepoPsqlImpl) FindAll(_ context.Context) ([]models.Account, error) {
```

Comment:
This is a small note for future improvement: We should implement pagination here instead of returning all the results because the system won't scale for big queries that will return many results.